### PR TITLE
Multiple domain support for Website Restrictions

### PR DESCRIPTION
**Pull request by JoRoRadar**

### CODE REVIEW

File: docs/faqs.mdx [hunk 1]
```diff
@@ -165,7 +165,7 @@ Yes. For more information, see the [Vulnerability Disclosure Policy](/vulnerabil
 - Use [Publishable keys](/sdk#authentication), which are restricted in scope in client-side code. Never use [Secret keys](/api#authentication), which can read or write any data.
 
 #### Website Restrictions
-Additionally, you can restrict which websites can use your publishable API keys. You can add or modify available domains on the [settings](https://radar.com/dashboard/settings/#website-restrictions) page.
+Additionally, you can restrict which websites can use your publishable API keys. You can add or modify available domains on the [settings](https://radar.com/dashboard/settings/#website-restrictions) page; entries into the field are comma-delimited.
```
Comment (brettguenther) [inline docs/faqs.mdx:168]: . Multiple domains should be input as a comma-delimited list.

**Comment by nickpatrick** [comment]
imo, better to fix this by making the field label and placeholder clearer in the dashboard. Field label could be `Live origins (comma-separated)` and placeholder could be`yourdomain1.com,yourdomain2.com`

**Comment by kochis** [comment]
This is selectable input, not comma separated.

image

**Comment by JoRoRadar** [comment]
@kochis - Ooops! Thanks for the catch! I'll update to say something to the effect of a selectable input.

**Review by jsani-radar — APPROVED** [review]

**Pull request merged**